assets/js/modules/adsense/index.js

```diff
@@ -54,7 +54,7 @@ let isAdBlockerActive = () => {};
 export const registerStore = ( registry ) => {
 	registerDataStore( registry );
 	// TODO: fix hack
-	isAdBlockerActive = () => registry.select( STORE_NAME ).isAdBlockerActive();
+	isAdBlockerActive = () => registry.__experimentalResolveSelect( STORE_NAME ).isAdBlockerActive();
```

Comment:
Issue to fix this hack is here: https://github.com/google/site-kit-wp/issues/2749